client/app/page.tsx

```diff
@@ -89,18 +87,19 @@ export default function Home() {
                             )}
                         />
 
-                        <div style={{ display: 'flex', justifyContent: 'center' }}>
-                            <Button asChild style={{ marginTop: '10px' }}>
-                                <Link href={isHostLogin ? '/host' : '/competitor'}>Login</Link>
-                            </Button>
+                        <div className="flex justify-center">
+                            <Button className="mt-2 w-full">Login</Button>
                         </div>
                     </form>
                 </Form>
 
                 {message && <p>{message}</p>}
-                <button className="login-view" onClick={toggleHostLogin}>
-                    {isHostLogin ? 'Login As Competitor' : 'Login As Host'}
-                </button>
+            </div>
+
+            <div className="height-20 flex items-center justify-center">
+                <a href="/leaderboard" className="cursor-pointer text-blue-500 hover:underline">
+                    Leaderboard
+                </a>
```

Comment:
This should not be done as an anchor tag.  It should be a Button with a Link inside of it, as linked in the original comment.